cmd/checks_windows.go

```diff
@@ -407,40 +407,39 @@ func securityPolicy(keyName, keyValue string) (bool, error) {
 	re := regexp.MustCompile("(?m)[\r\n]+^.*" + keyName + ".*$")
 	output := strings.TrimSpace(string(re.Find([]byte(seceditOutput))))
 	if output == "" {
-		return false, errors.New("securitypolicy item not found")
+		return false, errors.New("SecurityPolicy item not found")
 	}
 	if keyName == "NewAdministratorName" || keyName == "NewGuestName" {
 		// These two are strings, not numbers, so they have ""
 		desiredString = keyName + " = " + keyValue
 	} else if keyName == "MinimumPasswordAge" ||
 		keyName == "MinimumPasswordLength" ||
 		keyName == "LockoutDuration" ||
-		keyName == "ResetLockoutCount" {
+		keyName == "ResetLockoutCount" ||
+		keyName == "MaximumPasswordAge" ||
+		keyName == "LockoutBadCount" {
 		// Fields where the arg should be X or higher (up to 999)
-		intKeyValue, err := strconv.Atoi(keyValue)
+		intLow, err := strconv.Atoi(keyValue)
 		if err != nil {
 			failPrint(keyValue + " is not a valid integer for SecurityPolicy check")
 			return false, errors.New("Invalid keyValue")
 		}
-		for c := intKeyValue; c <= 999; c++ {
-			desiredString = keyName + " = " + strconv.Itoa(c)
-			if output == desiredString {
-				return true, err
+		var result1, _ = strconv.Atoi(strings.Split(output, " = ")[1])
+		if optValue != "" {
+			intHigh, err := strconv.Atoi(optValue)
+			if err != nil {
+				failPrint(optValue + " is not a valid integer for SecurityPolicy check")
```

Comment:
I'm concerned about this since, from my understanding, it'll print this message at runtime. Obviously, we don't want to tell the user that there's an issue with a `SecurityPolicy` check, so it's probably best to just `return err` here. 

I'll note that this isn't a best practice in terms of Go in general (rather, programming in general), but there's nothing we can do about it. Handling errors and keeping information from the client do not go hand in hand.

Reply:
Personally, I don't see the issue. Because what it's saying is, if you do have a value within `optValue` (`optValue != ""`), and converting it errors, then either A) remove `optValue`, or B) fix your `optValue`. 
If you look at the line below (line 432), it will return with the error of an invalid `optValue`. if there simply is no `arg3`, however, the system will just score `if result == arg2`, as per stated in the documentation for these changes.

I do not personally see an issue from my understanding of how my code works (and how it works in testing), but if I missed something specific, I'd be happy to make the changes.

Reply:
It works, yes, but it would be preferable if you didn't `failPrint` here. Other than that, this PR looks good.